datasets/logiqa_en/logiqa_en.py

```diff
@@ -0,0 +1,160 @@
+"""LogiQA, which is sourced from expert written questions for testing human logical reasoning. It consists of 8,678 QA instances, covering multiple types of deductive reasoning."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@misc{liu2020logiqa,
+      title={LogiQA: A Challenge Dataset for Machine Reading Comprehension with Logical Reasoning},
+      author={Jian Liu and Leyang Cui and Hanmeng Liu and Dandan Huang and Yile Wang and Yue Zhang},
+      year={2020},
+      eprint={2007.08124},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+LogiQA, which contains 8,678 paragraph-question pairs, each with four candidate answers. The dataset is sourced from publically available logical examination papers for reading comprehension, which are designed by domain experts for evaluating the logical reasoning ability and test participants.
+"""
+_HOMEPAGE = "https://github.com/lgw863/LogiQA-dataset"
+_LICENSE = "N/A"
+
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {"default": "https://github.com/lgw863/LogiQA-dataset/archive/master.zip"}
+
+
+class LogiQAEn(datasets.GeneratorBasedBuilder):
+    """LogiQA, which is sourced from expert written questions for testing human logical reasoning. It consists of 8,678 QA instances, covering multiple types of deductive reasoning."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    # BUILDER_CONFIGS = [
+    #     datasets.BuilderConfig(name="first_domain", version=VERSION, description="This part of my dataset covers a first domain"),
+    #     datasets.BuilderConfig(name="second_domain", version=VERSION, description="This part of my dataset covers a second domain"),
+    # ]
+    #
+    # DEFAULT_CONFIG_NAME = "first_domain"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "answer": datasets.Value("string"),
```

Comment:
This should be a `ClassLabel` feature:
```suggestion
                "answer": datasets.ClassLabel(names=["a", "b", "c", "d"]),
```

Reply:
Done!